chains/8453.json

```diff
@@ -834,6 +834,11 @@
         "isContract": true,
         "name": "CHAINLINK_WELL_USD"
     },
+    {
+        "addr": "0xc15d9944dAefE2dB03e53bef8DDA25a56832C5fe",
+        "isContract": true,
+        "name": "CHAINLINK_stkWELL_USD"
```

Comment:
better to have a special edge case for using `CHAINLINK_WELL_USD` feed for stkWell market (in the proposal script)